monkestation/code/modules/security/code/holographic_handcuffs.dm

```diff
@@ -0,0 +1,66 @@
+
+/**
+ * Handcuffs
+ * the handcuffs themselfes should be un-obtainable, /used version is applied on our actual target
+ * as strong zipties, take 50% longer to handcuff someone with
+ */
+
+/obj/item/restraints/handcuffs/holographic
+	name = "holographic energy field"
+	desc = "A weirdly solid holographic field... how did you get this? this item gives you the permission to scream at coders."
+	icon_state = "handcuffAlien"
+	lefthand_file = 'icons/mob/inhands/equipment/security_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/security_righthand.dmi'
+	breakouttime = 45 SECONDS
```

Comment:
Way too long for the free infinite handcuffs, should be 30 second or less

Reply:
copy comment, look below for actual conversation